packages/@okta/vuepress-site/docs/guides/set-up-token-exchange/main/index.md

```diff
@@ -0,0 +1,345 @@
+```
+
+**Access token decoded**
+
+To check the returned access token payload, you can copy the value and paste it into any JWT decoder (for example: https://jwt.io/). Then, verify that the scope claim (`scp`) and the audience claim (`aud`) are correct. The audience should match the default authorization server audience.
+
+```json
+{
+  "ver": 1,
+  "jti": "AT.nmEEZjEgPk62Us10J5aG4kUfg7DWoLIExSotYJA-rJ4",
+  "iss": "https://{yourOktaDomain}/oauth2/default",
+  "aud": "com.api.atko",
+  "iat": 1675220791,
+  "exp": 1675224391,
+  "cid": "0oa9aw62CPUnXomeU9c5",
+  "uid": "00u930ooYXgBltqbs9c5",
+  "scp": [
+    "api:access:read",
+    "api:access:write"
+  ],
+  "auth_time": 1675220791,
+  "sub": "user@example.com"
+}
+```
+
+## Trusted servers
+
+You can perform token exchange within a single custom authorization server or between other custom authorization servers under the same Okta tenant. The example discussed above is a token exchange within a single custom authorization server. When you want to perform token exchange between custom authorization servers within the same tenant, you need to make the authorization server that issued the subject token as trusted under the authorization server against which the token exchange request is made.
+
+A trusted server handles authenticated requests after the application obtains an access token. The incoming subject token (access token) is used to evaluate a subject.
+
+The following example walks you through how to set up a trusted server, scopes, access policy, and rule. Then, you can make the token exchange request with the trusted server.
+
+### Add a trusted server
+
+Add the custom authorization server that you used in the previous flow as a trusted server of the authorization server that handles the token exchange in the trusted server flow.
+
+1. In the Admin Console, go to **Security** > **API**.
+1. Select the edit (pencil) icon on the right of the custom authorization server that you want to associate a trusted server with for this example.
+1. In the **Trusted servers** section, click **Add Server**.
+1. In the **Search** field, enter the name of the authorization server that you used in the token exchange flow previously. Matching results appear in a list. If more than 20 results appear, you can click **Show more**.
+1. Click **Add** beside the authorization server, and then click *Done**. The authorization server appears in the **Trusted servers** section.
+
+### Create an access policy and rule
+
+Create an access policy and rule so that the service app can access API2 using a different authorization server than the one that granted the original access token for the user.
+
+1. Select the Access Policies tab, and then click Add New Access Policy to add a policy that allows the service app to access API2.
+1. In the Add Policy dialog that appears, enter the following:
+
+   * **Name:** Enter a name for the new access policy (**Access API2**).
+   * **Description:** Enter a description.
```

Comment:
Match this to whatever format you decide for the **Create access policies and rules** section.